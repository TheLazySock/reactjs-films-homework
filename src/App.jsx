import React from 'react';

import Signature from './components/Signature';


const App = () => (
  <React.Fragment>
    <Signature name="Uladzuslau Piatrenka" />
  </React.Fragment>
);

export default App;
