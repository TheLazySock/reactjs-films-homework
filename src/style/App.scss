.mainContainer {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  height: 100%;
  min-height: 100vh;
  background: #f6f6f6;
}
