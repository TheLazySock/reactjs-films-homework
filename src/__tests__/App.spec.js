import React from 'react';
import renderer from 'react-test-renderer';

import App from '../App.jsx';


describe('App', () => {
  it('renders correctly', () => {
    const output = renderer.create(<App />);
    expect(output).toMatchSnapshot();
  });
});
