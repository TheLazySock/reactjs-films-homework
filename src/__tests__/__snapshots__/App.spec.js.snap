// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="background"
>
  <h3
    className="name"
  >
    Uladzuslau Piatrenka
  </h3>
</div>
`;
